lib/assesments/assessment_2/fi_a2_bilal0x01/screens/login/widgets/logo_bilal0x01.dart

```diff
@@ -0,0 +1,18 @@
+part of '../login_screen_bilal0x01.dart';
+
+class _LogoBilal0x01 extends StatelessWidget {
```

Comment:
If a widget is just 5 lines of code, and isn't being used at multiple places why break it down into a seperate wideget?